### ISSUE TITLE
Tensor.masked_fill_(mask, value)  missing

### ISSUE THREAD

**Issue by toolgood** [dotnet/TorchSharp#872]
Tensor. masked_ fill_ And Tensor.masked_fill is two different methods

``` py
import torch

torch.manual_seed(2022)

tensor = torch.randn(4, 4)
tensor2 = torch.randn(4, 4)
mask = torch.zeros(4,4)

# build mask
for index, data in enumerate(range(3, -1, -1)):
    mask[index, data: ] = 1
# convert bool
mask = mask.bool()
tensor.masked_fill_(mask, -float('inf'))
print(tensor)
tensor2.masked_fill(mask, -float('inf'))
print(tensor2)
```
out
```
tensor([[-0.9788, -1.5154, -0.8222,    -inf],
        [ 0.0716, -0.0872,    -inf,    -inf],
        [ 0.2230,    -inf,    -inf,    -inf],
        [   -inf,    -inf,    -inf,    -inf]])
tensor([[-1.1104,  1.1890,  1.3730,  0.4915],
        [ 0.3579, -0.1685, -0.8579, -1.0574],
        [ 0.8460,  1.5126,  0.0289, -0.1584],
        [ 1.8371,  0.6688, -1.7776,  0.0020]])
```

**Comment by toolgood** [comment]
torch.nn.functional.interpolate  missing

**Comment by toolgood** [comment]
torch.softmax(Tensor tensor, long dim)   missing
torch.gather(input, dim, index, *, sparse_grad=False, out=None)    missing
torch.jit.annotate(the_type, the_value)   missing
torch.triu(input, diagonal=0, *, out=None)    missing

**Comment by NiklasGustafsson** [comment]
> torch.nn.functional.interpolate missing

I find it in [Vision.cs, line 209[( https://github.com/dotnet/TorchSharp/blob/2230dec43027443201a5bdced240e5f11ee71dc7/src/TorchSharp/NN/Vision.cs#L209)

Is there another version you are looking for?

**Comment by NiklasGustafsson** [comment]
> torch.softmax(Tensor tensor, long dim) missing

torch.nn.functional.softmax() is present, but torch.special.softmax() is missing. I presume it's the latter you're looking for?

**Comment by toolgood** [comment]
> torch.nn.functional.softmax() is present, but torch.special.softmax() is missing. I presume it's the latter you're looking for?

yes

**Comment by toolgood** [comment]
> > torch.nn.functional.interpolate missing
> 
> I find it in [Vision.cs, line 209[( https://github.com/dotnet/TorchSharp/blob/2230dec43027443201a5bdced240e5f11ee71dc7/src/TorchSharp/NN/Vision.cs#L209)
> 
> Is there another version you are looking for?

Thank you. I was wrong.

**Comment by toolgood** [comment]
> torch.nn.functional.softmax() is present, but torch.special.softmax() is missing. I presume it's the latter you're looking for?

yes
